test/run.d

```diff
@@ -56,6 +63,10 @@ Options:
     // allow overwrites from the environment
     resultsDir = environment.get("RESULTS_DIR", resultsDir);
     hostDMD = environment.get("HOST_DMD", "dmd");
+    unitTestRunnerCommand = resultsDir.buildPath("unit_test_runner");
+
+    if (runUnitTests)
+        return spawnProcess(unitTestRunnerCommand ~ args).wait();
```

Comment:
How do you ensure that this binary actually exists?
`ensureToolsExists` hasn't been called yet.

Reply:
Also to avoid the need for `--` e.g save `args` before doing the `args.popFront` on L60.

Reply:
> How do you ensure that this binary actually exists? `ensureToolsExists` hasn't been called yet.

That's a good point. I'll fix that.

> Also to avoid the need for `--` e.g save `args` before doing the `args.popFront` on L60.

That won't work because of the conflict with the `-f` flag.

Reply:
Fixed (ensuring the tool exists).